be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -81,7 +81,7 @@ void build_rowset_meta_with_spec_field(RowsetMeta& rowset_meta,
                                        const RowsetMeta& spec_rowset_meta) {
     rowset_meta.set_num_rows(spec_rowset_meta.num_rows());
     rowset_meta.set_total_disk_size(spec_rowset_meta.total_disk_size());
-    rowset_meta.set_data_disk_size(spec_rowset_meta.total_disk_size());
+    rowset_meta.set_data_disk_size(spec_rowset_meta.data_disk_size());
```

Comment:
We should check impact.

Reply:
The impact has already been checked. In old version, total_disk_size=data_disk_size. Index size is not considered at all. This PR we change it to total_disk_size=data_disk_size+index_size.